views/meetings/index.view.php

```diff
@@ -54,6 +53,12 @@ class="border-b border-neutral-200 font-medium dark:border-black/20">
                             <?= htmlspecialchars($meeting['topic']) ?>
                     </a>
                 </td>
+                <td class="<?= $table_row_style ?>">
+                   <p>
+                        <?= htmlspecialchars( $meeting['meeting_datetime']); ?>
```

Comment:
Можно отформатировать дату в нужный формат, убрать отображение секунд (они всегда будут 00)
```suggestion
                        <?= htmlspecialchars( date_format(date_create($meeting['meeting_datetime']), 'Y-m-d H:i')); ?>
```